src/components/search-by-concepts/search-by-concepts.component.tsx

```diff
@@ -1,73 +1,347 @@
+    onOrBefore,
+    operator,
+    operatorValue,
+    timeModifier,
+  ]);
+
+  useEffect(() => {
+    if (resetInputs) {
+      handleResetInputs();
+    }
+  }, [resetInputs]);
+
+  const handleLastDaysAndMonths = () => {
+    if (lastDays > 0 || lastMonths > 0) {
+      const onOrAfter = moment()
+        .subtract(lastDays, "days")
+        .subtract(lastMonths, "months")
+        .format();
+      setOnOrBefore(onOrAfter);
+    }
+  };
+
+  const handleDates = (dates: Date[]) => {
+    setOnOrAfter(moment(dates[0]).format());
+    setOnOrBefore(moment(dates[1]).format());
+  };
+
+  const handleResetInputs = () => {
+    setConcept(null);
+    setLastDays(0);
+    setOnOrAfter("");
+    setOnOrBefore("");
+    setLastMonths(0);
+    setOperatorValue(0);
+    setOperator("LESS_THAN");
+    setTimeModifier("ANY");
+  };
+
+  const handleInputValues = () => {
+    handleLastDaysAndMonths();
+    const observations: Observation = {
+      modifier: "",
+      onOrAfter: onOrAfter,
+      onOrBefore: onOrBefore,
+      operator1: operator,
+      question: concept.uuid,
+      timeModifier: timeModifier,
+      value1: operatorValue > 0 ? operatorValue.toString() : "",
+    };
+    const dataType = types[concept.hl7Abbrev];
+    const params = { [dataType]: [] };
+    Object.keys(observations).forEach((key) => {
+      observations[key] !== ""
+        ? params[dataType].push({
+            name:
+              key === "modifier"
+                ? ["CWE", "TS"].includes(concept.hl7Abbrev)
+                  ? "values"
+                  : "value1"
+                : key,
+            value:
+              key === "modifier" && ["CWE", "TS"].includes(concept.hl7Abbrev)
+                ? [observations[key]]
+                : observations[key],
+          })
+        : "";
+    });
+    setSearchParams(composeJson(params));
+    setQueryDescription(queryDescriptionBuilder(observations, concept.name));
+  };
 
-export const SearchByConcepts: React.FC = () => {
   return (
-    <div>
-      Search By Concepts
-      <Grid>
-        <Column>
-          <Search
-            closeButtonLabelText="Clear search input"
-            id="search-1"
-            labelText="Search"
-            onChange={(event) => getConcepts(event.target.value)}
-            onKeyDown={function noRefCheck() {}}
-            size="lg"
-          />
-        </Column>
-        <Column sm={2} md={{ span: 2, offset: 1 }}>
-          <Dropdown
-            helperText="This is some helper text"
-            id="default"
-            items={[
-              {
-                id: "option-0",
-                text: "Lorem, ipsum dolor sit amet consectetur adipisicing elit.",
-              },
-              {
-                id: "option-1",
-                text: "Option 1",
-              },
-            ]}
-            label=""
-            titleText=""
-          />
+    <div className={styles.container}>
+      <p className={styles.heading}>Search By Concepts</p>
```

Comment:
Can't we use carbon classes directly like this?

```suggestion
      <p className="productive-heading-02">Search By Concepts</p>
```

Reply:
I tried it doesn't work.